ffv1.md

```diff
@@ -975,6 +978,8 @@ FFV1 SHOULD use `V_FFV1` as the Matroska `Codec ID`. For FFV1 versions 2 or less
 
 ## Frame
 
+A `Frame` is the representation of video signals in the spatial domain. The whole `Frame` is provided by the underlaying container.
```

Comment:
same comment as below, that perhaps the definition of Frame from the Definitions section should be moved here, so as not to twice Frame twice using separate language.